src/components/Karma/CircleGradient.tsx

```diff
@@ -30,7 +30,8 @@ export const CircleGradient = (props: {
         height: props.size,
         justifyContent: 'center',
         alignItems: 'baseline',
-        borderRadius: 9999
+        borderRadius: 9999,
+        position: 'relative'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `position: 'relative'` style property has been added. This could potentially affect the positioning of child elements within this component. Ensure that this change does not unintentionally disrupt the layout of your application.
```diff
- borderRadius: 9999
+ borderRadius: 9999,
+ position: 'relative'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->